src/cider/nrepl/middleware/debug.clj

```diff
@@ -139,10 +143,10 @@
 ;;; `wrap-debug` receives an initial message from the client, stores
 ;;; it in `debugger-message`, and `breakpoint` answers it when asking
 ;;; for input.
-(def debugger-message
```

Comment:
I'm fine with this change, but I'd rather leave it to a separate commit (unless it has some relation to this feature).